Cargo.toml

```diff
@@ -31,6 +31,7 @@ serde_yaml = "0.8.8"
 base64 = "0.10.1"
 hex = "0.3.2"
 
+bech32lib = {version = "0.7.1", package="bech32" }
```

Comment:
Is it called bech32lib?

Reply:
I'm calling the internal module `bech32`, so I needed to rename the external lib.